examples/hello/app.py

```diff
@@ -1,7 +1,7 @@
-from spin_sdk.http import simple
-from spin_sdk.http.simple import Request, Response
+from spin_sdk.http import IncomingHandler
```

Comment:
Consider combining this line with the one below.

Reply:
done!